src/configuration-requestor.ts

```diff
@@ -0,0 +1,31 @@
+import { IConfigurationStore } from './configuration-store/configuration-store';
+import { IHttpClient } from './http-client';
+import { BanditParameters, Flag } from './interfaces';
+
+// Requests AND stores flag configurations
+export default class ConfigurationRequestor {
+  constructor(
+    private readonly httpClient: IHttpClient,
+    private readonly flagConfigurationStore: IConfigurationStore<Flag>,
+    private readonly banditConfigurationStore: IConfigurationStore<BanditParameters> | undefined,
+  ) {}
+
+  async fetchAndStoreConfigurations(): Promise<void> {
+    const configResponse = await this.httpClient.getUniversalFlagConfiguration();
+    if (!configResponse?.flags) {
+      return;
+    }
+
+    await this.flagConfigurationStore.setEntries(configResponse.flags);
+    if (Object.keys(configResponse.bandits ?? {}).length) {
+      // TODO: different polling intervals for bandit parameters
```

Comment:
I imagine doing this will become very important soon (to not have bandits double the requests to our CDN), but for now, we can just follow what is done in our [Java](https://github.com/Eppo-exp/java-server-sdk/blob/main/src/main/java/com/eppo/sdk/helpers/ConfigurationStore.java#L114) and [Python](https://github.com/Eppo-exp/python-sdk/blob/main/eppo_client/configuration_requestor.py#L67) SDKs. Lef the TODO in though so we don't forget about it!